hepcrawl/testlib/celery_monitor.py

```diff
@@ -81,7 +89,12 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             timeout=self.monitor_timeout,
             wakeup=True,
         )
-        self._wait_for_results(events_iter)
+
+        try:
+            self._wait_for_results(events_iter)
+        except SocketTimeout:
+            pass
```

Comment:
Shell we log something for this particular case?

Reply:
I don't think so, as what we want is just to wait see if there are any results sent, and if not, just return empty. It's not a non-expected case.